### ISSUE TITLE
scipy.optimize.fmin_slsqp on f7

### ISSUE THREAD

**Issue by nikohansen** [numbbo/coco#2301]
[`scipy.optimize.fmin_slsqp`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.fmin_slsqp.html) is one of the solvers in the upcoming `example_experiment_complete.py` script. Run with a budget multiplier of 40, we see 38 sweeps over the `bbob` suite, 34 of them done only on the Step Ellipsoid f7, the last three of them only in 40D (the algorithm terminates, if I am not mistaken, always after one iteration with $D+1$ evaluations and we do 2 additional evaluations, hence the budget 40 x D is exhausted after 38 sweeps).

It would be nice to tweak the solver options such that the behavior becomes more reasonable also on f7.

The above repetitions "mess up" some of the median runtime crosses, that is, the example is good test when fixing #2287.

**Comment by olafmersmann** [comment]
I think `fmin_slsqp` fails on $f7$ because the (approximated) gradient is zero almost everywhere which means it terminates immediately. Not sure you can fix that by changing some parameter.

**Comment by nikohansen** [comment]
Agreed, the chances are not too great, but I still would want to have look.
